UnitTests/CommonTestUtils/TempAppSettingsAttribute.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using GitCommands;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+
+namespace CommonTestUtils
+{
+    [AttributeUsage(AttributeTargets.Assembly)]
+    public sealed class TempAppSettingsAttribute : Attribute, ITestAction
```

Comment:
Maybe?
```suggestion
    public sealed class TestAppSettingsAttribute : Attribute, ITestAction
```

Reply:
Yes, will rename. The name `TempAppSettingsAttribute` was chosen with the expectation that the `AppSettings.SettingsFilePath` needed to be tinkered.